.gitignore

```diff
@@ -12,5 +12,9 @@ gui_layout.zig.zon
 
 test.png
 
+# editor files
+.vimrc
```

Comment:
Shouldn't it ignore all `**/.vimrc`? You had the .vimrc in a subfolder in the other PR.

Reply:
Nope, since the path is not specified it applies to all filenames.